duecredit/injections/mod_sklearn.py

```diff
@@ -32,3 +32,45 @@ def inject(injector):
         publisher={JMLR. org}
         }
     """), description="Machine Learning library")
+
+    # sklearn.cluster.affinity_propagation_
+    injector.add('sklearn.cluster.affinity_propagation_', None, Doi('10.1126/science.1136800'),
+                 description="Affinity propagation clustering algorithm", tags=['reference'])
+
+    # sklearn.cluster.bicluster
+    injector.add('sklearn.cluster.bicluster', 'SpectralCoclustering', Doi('10.1.1.140.301'),
+                 description="Spectral Coclustering algorithm", tags=['reference'])
+    injector.add('sklearn.cluster.bicluster', 'SpectralBiclustering', Doi('10.1.1.135.1608'),
+                 description="Spectral Biclustering algorithm", tags=['reference'])
```

Comment:
what was your decision on  "reference" vs "implementation"?  I guess we might need to adjust the documentation there.  I thought that "reference" would correspond to the "reference implementation", i.e. the one authors of the method implemented or "endorsed".  I guess for those above (all?) are just regular "implementation"s